docs/sources/whatsnew/whats-new-in-v11-1.md

```diff
@@ -0,0 +1,171 @@
+
+![Stat percent change color options](/media/docs/grafana/gif-grafana-11-1-stat-percent-change-color-options.gif)
+
+### XY Chart is GA
+
+<!-- Leon Sorokin -->
+
+_Generally available in all editions of Grafana_
+
+We’re promoting XY chart out of public preview and into general availability.
+
+XY charts provide a way to visualize arbitrary x and y values in a graph so that you can easily show the relationship between two variables. XY charts are typically used to create scatter plots. You can also use them to create bubble charts where field values determine the size of each bubble:
+
+![An xy chart showing height weight distribution](/media/docs/grafana/panels-visualizations/screenshot-xy-charts-v11.0.png)
+
+Over the past several months we’ve introduced multiple enhancements to the visualizations like auto mode, which now handles most scenarios that previously required manual configuration. Additionally, we've added better control over point styling and further improved performance. We’re excited to include XY chart as a first class citizen in the core Grafana visualization library. To learn more about the panel, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/xy-chart/).
+
+To use the new XY chart visualization, you must first enable the `autoMigrateXYChartPanel` [feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/).
+
+## Alerting
+
+<!-- TO DO - Confirm with Alerting folks that everything available in OSS is also available in Enterprise except for Rule specific silences with permissions and Rule specific silences with RBAC-->
+
+### Re-designed settings page for Alerting
+
+<!-- Gilles De Mey -->
+
+_Generally available in all editions of Grafana_
+
+The new settings page provides you with a holistic view of where Grafana-managed alert instances are forwarded.
+
+- Manage which Alertmanagers receive alert instances from Grafana-managed rules without navigating and editing data sources.
+- Manage version snapshots for the built-in Alertmanager, which allows administrators to roll back unintentional changes or mistakes in the Alertmanager configuration.
+- There is also a visual diff that compares the historical snapshot with the latest configuration to see which changes were made.
+
+{{< figure src="/media/docs/alerting/alert-settings.png" alt="Alert settings page" >}}
+
+### Alerting template selector
+
+<!-- Sonia Aguilar -->
+
+_Generally available in all editions of Grafana_
```

Comment:
@soniaAguilarPeiron - Can you please confirm that this is available in both OSS **and** Enterprise? It was only listed as being available in Cloud and OSS on the original Cloud entry.

Reply:
Available in both OSS and Enterprise

Reply:
Thanks very much for checking on these!